src/librustdoc/test.rs

```diff
@@ -262,17 +262,19 @@ fn runtest(test: &str, cratename: &str, cfgs: Vec<String>, libs: SearchPaths,
         control.after_analysis.stop = Compilation::Stop;
     }
 
-    let res = panic::catch_unwind(AssertUnwindSafe(|| {
```

Comment:
I think this reads better having the panic + closure in a variable and then later matching the variable rather than doing both at once.